bin/node/testing/src/bench.rs

```diff
@@ -287,15 +290,29 @@ impl BenchDb {
 			let signed = self.keyring.sign(
 				CheckedExtrinsic {
 					signed: Some((sender, signed_extra(0, node_runtime::ExistentialDeposit::get() + 1))),
-					function: Call::Balances(
-						BalancesCall::transfer(
-							pallet_indices::address::Address::Id(receiver),
-							match block_type {
-								BlockType::RandomTransfers(_) => node_runtime::ExistentialDeposit::get() + 1,
-								BlockType::RandomTransfersReaping(_) => 100*DOLLARS - node_runtime::ExistentialDeposit::get() - 1,
-							}
-						)
-					),
+					function: match block_type {
+						BlockType::RandomTransfers(_) => {
+							Call::Balances(
+								BalancesCall::transfer(
+									pallet_indices::address::Address::Id(receiver),
+									node_runtime::ExistentialDeposit::get() + 1,
+								)
+							)
+						},
+						BlockType::RandomTransfersReaping(_) => {
+							Call::Balances(
+								BalancesCall::transfer(
+									pallet_indices::address::Address::Id(receiver),
+									100*DOLLARS - node_runtime::ExistentialDeposit::get() - 1,
```

Comment:
Is this right? Copied from what was originally in master, but without much more context, I assume that:

* Sender starts with 100 dollars
* we want to kill the sender account

So we should send `100 - (ed - 1)` (note the parenthesis)

It seems here we dont send enough to kill the account? unless fees are also taken?

Reply:
I question this because I found that `RandomTransferReaping` has a faster benchmark than `RandomTransfers`.

Reply:
Probably got outdated (wasn't used in new version of benchmarks)

But also take in account transaction fee

Reply:
ive removed it for now, but it raises some red flags if we cannot explain why this is faster